draft-ietf-tls-esni.md

```diff
@@ -272,35 +280,21 @@ of the structure.  This hash is computed using the hash function
 associated with `suite`.
 
 encrypted_sni
-: The ClientESNIInner structure, AEAD-encrypted using cipher suite "suite" and
-the key generated as described below.
+: The serialized ClientHelloInner structure, AEAD-encrypted using
+cipher suite "suite" and the key generated as described below.
 {:br}
 
-For servers (in EncryptedExtensions), this extension contains the following
-structure:
 
-~~~
-   enum {
-       esni_accept(0),
-       esni_retry_request(1),
-   } ServerESNIResponseType;
+If the server accepts ESNI, then does not send this extension.
+If it rejects ESNI, then it sends the following structure in
```

Comment:
This paragraph talks about sending a ESNIKeys that the client should use for retrying.

Therefore, maybe s/If it rejects ESNI/If the client sent an ESNI "encrypted_client_hello" extension using a key not available to the server/.